frontend/talentsearch/src/js/components/applicantProfile/ExperienceAndSkills.tsx

```diff
@@ -139,6 +139,12 @@ export const ExperienceAndSkills: React.FunctionComponent<
         description:
           "Heading for experience and skills page in applicant profile.",
       })}
+      cancelLink={{
+        children: intl.formatMessage({
+          defaultMessage: "Go back to my profile",
+          description: "Link text for button to return to user profile",
+        }),
+      }}
```

Comment:
You're going to need to update the `<ProfileFormFooter />` [on line 210](https://github.com/GCTC-NTGC/gc-digital-talent/blob/dd9a2915d67008f44763b236cd68f9cff807fdd4/frontend/talentsearch/src/js/components/applicantProfile/ExperienceAndSkills.tsx#L210) to accept a cancelLink as well.